server/memphis_handlers_functions_cloud.go

```diff
@@ -62,9 +62,29 @@ func (fh FunctionsHandler) GetFunctions(tenantName string) (models.FunctionsRes,
 	if err != nil {
 		return models.FunctionsRes{}, err
 	}
+	if len(functions) > 10 {
```

Comment:
This is not good, I told we are getting 10 in order to not reach the rate limits. What you are doing is to scan the entire repo and if you have more than 10 you return only 10. This is not good